challenges/objects-arrays.js

```diff
@@ -79,9 +111,10 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 {"animal_name":"Australian pelican","population":5,"scientific_name":"Pelecanus conspicillatus","state":"West Virginia"}];
 
 // The zoos need a list of all their animal's names converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
-let lowerCase = [];
-console.log(lowerCase); 
+let namesOnly = zooAnimals.map(element => {return element['animal_name'];});
+let lowerCase = namesOnly.map(element => {return element.toLowerCase();});
```

Comment:
No need for the additional `let` statement. And you may consider using `animalName` in lieu of `element` for readability.

`let lowerCase = zooAnimals.map((animalName) => {`
  `return animalName.animal_name.toLowerCase();`
`});`